zookeeper-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,47 @@
+registryItems:
+  applications:
+  - {zookeeper-scenario: nq 0}
+  instances:
+  - {zookeeper-scenario: 1}
+  operationNames:
+  - zookeeper-scenario: [/zookeeper/zookeeper-case, Zookeeper/exists, Zookeeper/delete]
+segmentItems:
+- applicationCode: zookeeper-scenario
+  segmentSize: nq 0
+  segments:
+  - segmentId: nq 0
+    spans:
+    - operationName: Zookeeper/exists
+      operationId: 0
+      parentSpanId: 0
+      spanId: 1
+      spanLayer: Cache
+      startTime: nq 0
+      endTime: nq 0
+      componentId: 58
+      componentName: ''
+      isError: false
+      spanType: Exit
+      peer: 'not null'
+      peerId: 0
+      tags:
+      - {key: db.type, value: Zookeeper}
+      - {key: path, value: /path}
+      - {key: watch, value: 'true'}
+    - operationName: Zookeeper/WatchedEvent/NodeDeleted
+      operationId: 0
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Unknown
+      startTime: nq 0
+      endTime: nq 0
+      componentId: 58
+      componentName: ''
+      isError: false
+      spanType: 'not null'
+      peer: 'not null'
+      peerId: 0
+      tags:
+      - {key: path, value: /path}
+      - {key: state, value: SyncConnected}
+      - {key: db.type, value: Zookeeper}
```

Comment:
The expected data miss the `/zookeeper/zookeeper-case` span

Reply:
The '/zookeeper/zookeeper-case' has added.